dfa/src/main/scala/dfa/DFA.scala

```diff
@@ -1,3 +1,17 @@
 package dfa // leave this line in the file
 
-// TODO: replace this comment with your implementation
+case class State(label: String)
+case class Transition(from: State, to: State, symbol: Char)
+
+class DFA(val states: Set[State], val start: State, val accept: Set[State], val transitions: Set[Transition]):
+    def accepts(input: String): Boolean = 
+        var currentState = start
+        for (x <- 0 to input.length()-1) {
```

Comment:
I did this part recursively, so it's interesting to see how you did it iteratively!